fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/Objects.kt

```diff
@@ -41,7 +42,7 @@ enum class Objects
       COIN.`object`,
       MESSAGE.`object` -> true
 
-      // Consensus and Popcha are for now the ones excluded from persistence
+      // Consensus, Popcha and Federation are for now the ones excluded from persistence
       else -> false
```

Comment:
Did you decide to not persist federation messages for a matter of time and it's gonna happen in a future PR or is there any other reason?

Reply:
Yes, for now it doesn't persist but it will happen in a future PR. We are still discussing about persistence and reconnection with the federation team.